packages/@expo/config-plugins/src/utils/Updates.ts

```diff
@@ -162,6 +162,10 @@ export function getUpdatesCheckOnLaunch(
     return 'NEVER';
   } else if (config.updates?.checkAutomatically === 'ON_LOAD') {
     return 'ALWAYS';
```

Comment:
it's weird that this has a non-matching name

Reply:
Almost as weird as `checkAutomatically` translating to `CHECK_ON_LAUNCH` in native code